oeps/oep-0011-bp-FED-technology.rst

```diff
@@ -160,6 +162,36 @@ Technology Selection
 
    You can find out more about Sass in the official `Sass documentation`_.
 
+10. **API calls should be made with edX Frontend Auth Client or Axios**
+   
+   **Rationale**: The `edX Frontend Auth Client`_ simplifies the process of
+   talking to edX APIs by using Axios inteceptors and handling JWT Cookie
+   authentication. It also provides React components to handle private routes
+   and should be used when possible. When making calls to non-edX APIs
+   Axios should be used to provide a consistent API.
+
+   The `fetch`_ API was considered but Axios was chosen for its more
+   intuitive API, particularly when handling with HTTP errors with rejected
+   promises.
+
+11. **Server-side content should be rendered with Django Templates**
```

Comment:
I guess server-side JavaScript is still at the review stage, is it too early to note that it's under consideration here?